src/main/java/co/cask/plugin/profiles/Logical.java

```diff
@@ -74,8 +74,8 @@ public void update(Object value) {
 
   @Override
   public void results(StructuredRecord.Builder builder) {
-    builder.set("true", countTrue);
-    builder.set("false", countFalse);
+    builder.set("positive", countTrue);
```

Comment:
it seems like true and false were correct, but the fields above were wrong.

Reply:
true and false will fail when you create an external hive table.

Reply:
so 'positive' and 'negative' is the intended behavior? Seems weird, but ok. Being forced to use a confusing name due to an unrelated bug isn't very great... should prioritize fixing that issue, it's been a recurring problem for many releasing.